build.sh

```diff
@@ -38,12 +39,13 @@ CMAKE_OPTS=""
 BUILD_TARGET="${MACHINE_ID:-'localhost'}"
 BUILD_VERBOSE="NO"
 CLONE_JCSDADATA="NO"
+CLONE_RRFSDATA="NO"
```

Comment:
@delippi  could we let the "-r" option default to Yes?  so, by default, they could have the rrfs tests available.

Reply:
@TingLei-NOAA , Well, how exactly does the COPY command work in the CMakeLists.txt file? Is it a standard `cp` or is it like `rsync` and only copy the things that don't already exist or have changed? I could update the COPY to an `rsync` command and then it might be okay to do this.

Otherwise, I kind of think it might be best to keep it as specifying `-r` means copy the data and excluding it means do not copy the data. It would only be needed during the initial compilation. This way when you recompile you don't need to specify the `-r` option again because you have already copied it. Otherwise, you would have to specify the option every time you recompile to not recopy the data and take more time than necessary to compile. The way it is now, if you forget to use the option the first time around you could just do something like `./build.sh -r -t hera` to copy the data. What do you think is best?

Reply:
@delippi  I got your points.  To avoid the repeated copying (the "copy" is to override existing files), we just need to   be careful not to use  "-f" option except for the first building from the command line options for the build.sh.  Second, in fact, for the changes only related to the codes, we won't need to run cmake/build.sh, just run make command.  In both situations, the data will not be copied again.    So, only the first time building of rdasapp will copy those test data, which I think we would expect.

Reply:
A correction,  run build.sh without -r option, the copying will still be re-done .  We can add some test to control this.

Reply:
Set the default to not copy those data,  now that will also decide if rrfs tests will be added.  If any changes in the cmake related files, the building process would re-run the configuration process, and maybe, will re-create the ctest set and might cause the new ctest sets excluding the rrfs tests.     Of course, all the above consideration could be dealt with more detailed control.  One is to use some rrfs_data_test_found options as in gdas.  
For being now, i think the  setup could be as it is now.

Reply:
Thanks for your input @TingLei-NOAA. I will keep everything as is for now. We should get this PR merged sooner rather than later. Let's work on these such items in a future PR where we can better fine tune our settings.